husky_directory/static/css/print.css

```diff
@@ -0,0 +1,5 @@
+/****** Print Styles ******/
+
+#wheader, .cblock, .rsummary, .mobileHeader, #footerMain, #footerMainNoPatch, .title, .btn-primary  {
+	display: none;
+}
```

Comment:
Please add a blank line at the end of this file (there are several instances of this; you can see the red warning circle in this github view)